scripts/bench.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Check if wallet and storage arguments are provided
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 <wallet> <storage>"
+    exit 1
+fi
+
+# Assign arguments to variables
+wallet=$1
+storage=$2
```

Comment:
Nit: this should probably be in double quotes, just in case the wallet is in a path with spaces

```suggestion
wallet="$1"
storage="$2"
```